microservices-connector/internal/controller/gmconnector_controller.go

```diff
@@ -510,11 +553,14 @@ func adjustConfigmap(ns string, hwType string, yamlMap *map[string]interface{},
 			if err == nil {
 				//check GMC config if there is specific namespace for embedding
 				altNs := getNsFromGraph(gmcGraph, TeiEmbedding)
-				if altNs != "" {
-					data["TEI_EMBEDDING_ENDPOINT"] = fmt.Sprintf("http://%s.%s.svc.cluster.local:%d", svcName, altNs, port)
-				} else {
-					data["TEI_EMBEDDING_ENDPOINT"] = fmt.Sprintf("http://%s.%s.svc.cluster.local:%d", svcName, ns, port)
+				if altNs == "" {
```

Comment:
I remember that this kind of changes have been modified in my previous PR,  do you update the latest code from main branch?

Reply:
there are more spot to be changed other than your PR